### DISCUSSION TITLE
`Sign in with GitHub` redirects to 127.0.0.1:4000/.oauth/authorize

### DISCUSSION THREAD

**Discussion by andry81** [lowlighter/metrics#1191] [Q&A] [answered]
In tag version said I can login with GitHub: https://github.com/lowlighter/metrics/releases/tag/v3.27

It does not work because tries to redirect into local address.

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

Yes sorry, I forgot to update the redirect url back to the live version 😅 
It's fixed now 👍 

Thanks for reporting it !

**Reply by andry81** [reply]
It does redirecting now ... infinitely. It prints `Redirecting...` and nothing happens.

**Reply by lowlighter** [reply]
Do you have any error in the web console by any chance ?
If not can you tell me which browser/os/device you're using ?

**Reply by andry81** [reply]
> You can now login through GitHub OAuth on [metrics.lecoq.io](https://metrics.lecoq.io/) to use your own read-only token (revokable anytime) and access more API intensive features that aren't enabled on the web instance by default!

How to use that read-only token to access additional API features? There said that I logged in but nothing else. Is not clear where to take that read-only token or how to use these features?

OS: Windows 7 x64
Browser: Mozilla Firefox

**Reply by andry81** [reply]
If try to revoke the OAuth within the GitHub and login again, then it hangs again. But if to reopen the link and login again, then it show that I am is logged in.

**Reply by lowlighter** [reply]
Maybe the JS redirection doesn't work properly, will try to look at it 👍
